discovery_engine/lib/assets/asset_manifest.json

```diff
@@ -2,34 +2,22 @@
   "assets": [
     {
       "id": "smbertVocab",
-      "url_suffix": "smbert_v0000/vocab.txt",
+      "url_suffix": "smbert_v0001/vocab.txt",
       "checksum": "ffb2398810fa977ee39e8a0103b07f9c2614f1f9825df53a2c99e71569e3451c",
       "fragments": []
     },
     {
       "id": "smbertModel",
-      "url_suffix": "smbert_v0000/smbert.onnx",
-      "checksum": "c07653a3cc568876071de157f0fb8ccc1a302197ddad9ead069a86140e19d913",
+      "url_suffix": "smbert_v0001/smbert-quantized.onnx",
+      "checksum": "83bf0604484231f2d2016aacb0b6e647ee65e1f29024c8c53b8a347e39b745db",
       "fragments": [
         {
-          "url_suffix": "smbert_v0000/smbert.onnx_11MB_00",
-          "checksum": "7f4d6242ef88ab6daaf1e1fa1f030345c4f4046585cd9576112aa5391f42a000"
+          "url_suffix": "smbert_v0001/smbert-quantized.onnx_11MB_00",
```

Comment:
Here we need the model without framents. Afaik team blue does not support downloading fragments in their script. the reason why it worked before was that smbert_v0000 was uploaded as a whole and in fragments. For now we also don't need fragments because we bundle the data in the app.